docs/source/sample_scripts/get_task_logs.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python2.7
+# -*- coding: utf-8 -*-
+
+from genestack_client import TaskLogViewer, make_connection_parser, get_connection
+
+# add extra arguments to the Genestack arguments parser for this script
+parser = make_connection_parser()
+parser.add_argument('--follow', action='store_true', help='Follow the logs\' output if the task is not done')
+parser.add_argument('--error', action='store_true', help='Display error logs instead of output logs')
+parser.add_argument('accession', help='Accession of the file for which to display the logs')
```

Comment:
Please add `metavar` to this field Example: https://github.com/genestack/python-client/blob/master/genestack_client/utils.py#L39